backend/controllers/authUserSearch.test.js

```diff
@@ -32,10 +32,15 @@ describe('Auth user search controller', () => {
     expect(res.json).toBeCalledWith(response)
   })
 
-  it('should return 400 if missing query', async () => {
-    await authUserSearch({ query: {} }, res)
+  it('should throw error if no results', async () => {
+    oauthApi.userSearch.mockReturnValue('')
+
+    await expect(authUserSearch({ query: { nameFilter: 'bob@joe.com' } }, res)).rejects.toThrow(
+      new Error('No results returned from search')
+    )
+  })
 
-    expect(res.json).toBeCalledWith('Enter a username or email address')
-    expect(res.status).toBeCalledWith(400)
+  it('should return 400 if missing query', async () => {
+    await expect(authUserSearch({ query: {} }, res)).rejects.toThrow(new Error('Missing name filter on search'))
```

Comment:
this is bit naff in that we have to compare the error `message` rather than the `error_description` which is used by the front end.  Suggestions appreciated :grin:

Reply:
Not sure, Maybe @jsrobertson has some thoughts?

Reply:
If anything I guess you could have a few helper functions that return errors? buildMissingNameFilterError= () => new Error('Missing name filter on search')